package.json

```diff
@@ -1,54 +1,96 @@
 {
   "name": "videojs-contrib-hls",
-  "version": "1.3.3",
-  "engines": {
-    "node": ">= 0.10.12"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:videojs/videojs-contrib-hls.git"
-  },
+  "description": "A video.js tech that plays HLS video on platforms that don't support it but have Flash",
+  "author": "Brightcove",
   "license": "Apache-2.0",
-  "scripts": {
-    "test": "grunt test",
-    "prepublish": "if [ -z \"$TRAVIS\" ]; then grunt; fi"
-  },
+  "version": "0.0.0",
```

Comment:
version got reset. Also, we try to keep the order of properties the same (name, version, description). This may have been fixed in a newer version of the generator.